<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join Attributes</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Target vector layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inShape"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Target join field</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inField"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Join data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QRadioButton" name="rdoVector">
          <property name="text">
           <string>Join vector layer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="joinShape"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QRadioButton" name="rdoTable">
          <property name="text">
           <string>Join dbf table</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="inTable">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolTable">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Join field</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="joinField"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labEncoding">
          <property name="text">
           <string>Encoding</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbEncoding"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output Shapefile</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="outShape">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolOut">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output table</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rdoMatch">
        <property name="text">
         <string>Only keep matching records</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rdoKeep">
        <property name="text">
         <string>Keep all records (including non-matching target records)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoTable</sender>
   <signal>clicked()</signal>
   <receiver>joinField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoVector</sender>
   <signal>clicked()</signal>
   <receiver>joinField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoVector</sender>
   <signal>toggled(bool)</signal>
   <receiver>joinShape</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>inTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
